src/test/ovmsconfig_test.cpp

```diff
@@ -286,13 +322,144 @@ TEST_F(OvmsParamsTest, hostname_ip_regex) {
+};
+
+TEST(OvmsConfigTest, positiveMulti) {
+    char* n_argv[] = {"ovms",
+        "--port", "44",
+        "--grpc_workers", "2",
+        "--grpc_bind_address", "1.1.1.1",
+        "--rest_port", "45",
+        "--rest_workers", "46",
+        "--rest_bind_address", "2.2.2.2",
+        "--grpc_channel_arguments", "grpc_channel_args",
+        "--file_system_poll_wait_seconds", "2",
+        "--sequence_cleaner_poll_wait_minutes", "7",
+        "--custom_node_resources_cleaner_interval", "8",
+        "--cpu_extension", "/ovms",
+        "--cache_dir", "/tmp/model_cache",
+        "--log_path", "/tmp/log_path",
+        "--log_level", "ERROR",
+
+        "--config_path", "/config.json"};
+    int arg_count = 31;
+    MockedConfig config;
+    config.parse(arg_count, n_argv);
+
+    EXPECT_EQ(config.port(), 44);
+    EXPECT_EQ(config.grpcWorkers(), 2);
+    EXPECT_EQ(config.grpcBindAddress(), "1.1.1.1");
+    EXPECT_EQ(config.restPort(), 45);
+    EXPECT_EQ(config.restWorkers(), 46);
+    EXPECT_EQ(config.restBindAddress(), "2.2.2.2");
+    EXPECT_EQ(config.grpcChannelArguments(), "grpc_channel_args");
+    EXPECT_EQ(config.filesystemPollWaitSeconds(), 2);
+    EXPECT_EQ(config.sequenceCleanerPollWaitMinutes(), 7);
+    EXPECT_EQ(config.resourcesCleanerPollWaitSeconds(), 8);
+    EXPECT_EQ(config.cpuExtensionLibraryPath(), "/ovms");
+    EXPECT_EQ(config.cacheDir(), "/tmp/model_cache");
+    EXPECT_EQ(config.logPath(), "/tmp/log_path");
+    EXPECT_EQ(config.logLevel(), "ERROR");
+
+    EXPECT_EQ(config.configPath(), "/config.json");
+}
+
+TEST(OvmsConfigTest, positiveSingle) {
+    char* n_argv[] = {
+        "ovms",
+        "--port",
+        "44",
+        "--grpc_workers",
+        "2",
+        "--grpc_bind_address",
+        "1.1.1.1",
+        "--rest_port",
+        "45",
+        "--rest_workers",
+        "46",
+        "--rest_bind_address",
+        "2.2.2.2",
+        "--grpc_channel_arguments",
+        "grpc_channel_args",
+        "--file_system_poll_wait_seconds",
+        "2",
+        "--sequence_cleaner_poll_wait_minutes",
+        "7",
+        "--custom_node_resources_cleaner_interval",
+        "8",
+        "--cpu_extension",
+        "/ovms",
+        "--cache_dir",
+        "/tmp/model_cache",
+        "--log_path",
+        "/tmp/log_path",
+        "--log_level",
+        "ERROR",
+
+        "--model_name",
+        "model",
+        "--model_path",
+        "/path",
+        "--batch_size",
+        "(3:5)",
+        "--shape",
+        "(3:5,5:6)",
+        "--layout",
+        "nchw:nhwc",
+        "--model_version_policy",
+        "setting",
+        "--nireq",
+        "2",
+        "--target_device",
+        "GPU",
+        "--plugin_config",
+        "pluginsetting",
+        "--stateful",
+        "--metrics_enable",
+        "--metrics_list",
+        "ovms_streams,ovms_other",
+        // "--idle_sequence_cleanup", "false",  // TOOD: cannot disable, change to --disable_idle_seq_cleanup?
```

Comment:
Do we have a task for this?

Reply:
It works with `--idle_sequence_cleanup=false`. Updated the unit test and removed todo.